repose-aggregator/services/datastore/impl/src/main/java/org/openrepose/core/services/datastore/impl/ehcache/EHCacheDatastore.java

```diff
@@ -66,6 +66,7 @@ public void put(String key, Serializable value) {
 
     @Override
     public void put(String key, Serializable value, int ttl, TimeUnit timeUnit) {
+        // todo: does not allow for eternal caching (need to either setEternal(true) or set ttl and tti to 0
```

Comment:
Or attempt to, anyway.

I think I added a story, but without being able to see the board, it's difficult to confirm. These are hard times with the JIRA freakout of 2015.